configuration/sitemaps.md

```diff
@@ -506,29 +529,33 @@ visibility=[Day_Time=="Morning", Day_Time=="Afternoon", Temperature>19]
 ```
 
 If any one of the comparisons is evaluated as `true`, the Item will be visible, otherwise it will be hidden.
-It is important to note, that it is not possible to decide visibility on more than one condition at the same time.
-The third example might be the visibility of a sprinkler control.
-The control will be visible if it is Morning *OR* if it is Afternoon *OR* if the temperature is above 19°C.
-To achieve more complex conditions, you will benefit from defining a helper Item and a rule to set it.
+It is important to note that visibility may only be determined based on one condition at a time.
 
-Valid comparison operators are:
+The third example might control the visibility of a lawn sprinkler control.
+The control will be visible if it is Morning *OR* if it is Afternoon *OR* if the temperature is above 19°C.
+To achieve more complex conditions, you will benefit from defining a helper Item and a [rule]({{base}}/configuration/rules-dsl.md) to set it.
 
-- equal to `==`, unequal to `!=`
-- smaller or equal to `<=`, bigger or equal to`>=`.
-- smaller than `<`, bigger than `>`
+<!-- TODO: Provide an example for the statement above or remove it.  It is not that helpful as it is now for people who are just getting aquainted with sitemaps -->
```

Comment:
Which?

Reply:
Now it is my turn to have confused you with a short comment ;-).

So the line above starts out, "To achieve more complex conditions, you will benefit from defining a helper Item..."

What is the meaning of "To achieve more complex conditions"?
What is a "helper Item"?

These seem like more advanced concepts and can be confusing as it is currently written.

I think I understand the first point - maybe it is trying to say, "If you are trying to do something more complicated than these simple examples, then you could define a helper Item..."

But I still don't know what a helper Item is.  A short simple example would really be good - otherwise I suggest we delete the whole idea (but that would be too bad, since then people would have to dig other places to find out how to do more complex things (but hey - they have to do that anyway for a lot of other stuff (Oops - nested parentheses again.))).

Reply:
Here's what the sentence intends to say: "If you are trying to do something more complicated than these simple examples, then you could define an additional Item which is set by a rule based on your more complex idea"

Upon writing this I agree to leave the whole sentence out.

Reply:
Actually, I quite like the idea of leaving "bread crumbs" for people who may be trying to do something more than just the basics.  Many times I find that all I need is a small hint about how to attack a more complex issue and I can take it from there.

How about we change the "To achieve more complex..." sentence to:

If you are trying to do something more complicated than these simple examples, then you could define an additional `Item` which is set by a `Rule`.
Rules have a rich set of features that accommodate more complex scenarios.